app/Schedule/View.elm

```diff
@@ -269,6 +274,16 @@ predictionColor minutesLate =
             Color.red
 
 
+arrival : Date -> Train -> Node Msg
+arrival date train =
+    trainMetadata ("arrives at" ++ arrivalTime train)
```

Comment:
It doesn't. Huh. Weird that it doesn't seem to make a difference, according to the screenshots.